MvvmCross/Core/Core/ViewModels/MvxViewModelExtensions.cs

```diff
@@ -50,7 +54,7 @@ public static void CallBundleMethod(this IMvxViewModel viewModel, MethodInfo met
             }
 
             // call method using named method arguments
-            var invokeWith = bundle.CreateArgumentList(parameters, viewModel.GetType().Name)
+            var invokeWith = bundle?.CreateArgumentList(parameters, viewModel.GetType().Name)
```

Comment:
Suddenly we get here and `bundle` can be null?  This seems like a contract violation since previously in our history `bundle` could never be null.